src/Common/CurrentMetrics.cpp

```diff
@@ -69,6 +69,8 @@
     M(PartsInMemory, "In-memory parts.") \
     M(MMappedFiles, "Total number of mmapped files.") \
     M(MMappedFileBytes, "Sum size of mmapped file regions.") \
+    M(AsyncDrainedConnections, "Number of connections drained asynchronously.") \
+    M(SyncDrainedConnections, "Number of connections drained synchronously.") \
```

Comment:
Also, I think, it would be interesting to check how many async drains in pool right now. And how many sync drains are executed at the moment.

Reply:
:+1: